integration/echo_test.sh

```diff
@@ -56,7 +56,7 @@ say "- setup ledger"
 ledger_init
 
 say "- echo test"
-echo_test
+#echo_test
```

Comment:
better add a TODO.

Reply:
Actually why is echo test disabled but auction test not? Echo test fails?

Reply:
none of these tests currently work, also as ecc is not yet updated. That said, i also would see let's not disable it here.  I actually forgot to mention to jordan that for ledger setup/teardown there are standalone scripts in `fabric/bin` (`ledger_init.sh` and `ledger_shutdown.sh`), so let's use to test ledger setup and teardown and leave the actual test-scripts untouched ..

Reply:
So that would basically mean I would have to copy both ledger_init.sh and ledger_shutdown.sh to the integration directory....so they can access "config"